### PR TITLE
PR: Remove CONF calls on widgets for issue reporter

### PR DESCRIPTION
**Pull request by goanpeca**
## Description of Changes

* [x] Wrote at least one-line docstrings (for any new functions)

### Issue(s) Resolved

Fixes #

### Affirmation

By submitting this Pull Request or typing my (user)name below,
I affirm the [Developer Certificate of Origin](https://developercertificate.org)
with respect to all commits and content included in this PR,
and understand I am releasing the same under Spyder's MIT (Expat) license.

I certify the above statement is true and correct: @goanpeca

### CODE REVIEW

**Comment by goanpeca** [comment]
This is part of moving CONF out of widgets, which should in general not care about what CONF is or if it exists at all.

File: spyder/app/mainwindow.py [hunk 1]
```diff
@@ -2715,6 +2715,7 @@ def report_issue(self, body=None, title=None, open_webpage=False):
         if body is None:
             from spyder.widgets.reporterror import SpyderErrorDialog
             report_dlg = SpyderErrorDialog(self, is_report=True)
+            report_dlg.set_color_scheme(CONF.get('appearance', 'selected'))
```
Comment (ccordoba12) [inline spyder/app/mainwindow.py]: This is not the only place where we create an instance of SpyderErrorDialog. There's also one in `spyder/plugins/console/plugin.py`.

Comment (goanpeca) [reply spyder/app/mainwindow.py]: I will fix it there! thanks!

Comment (goanpeca) [reply spyder/app/mainwindow.py]: Fixed @ccordoba12

**Pull request merged**